support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts

```diff
@@ -240,6 +240,7 @@ const POLLING_INTERVAL = 3000;
 const MAX_POLLS = 10;
 
 // ----- Functions ----- //
+
```

Comment:
Revert this change, this file hasn't actually changed so there's no need to clutter up the PR with it